### ISSUE TITLE
Official NuGet package for Nini

### ISSUE THREAD

**Issue by roman-yagodin** [bmatzelle/nini#4]
There are some "unofficial" NuGet packages for Nini, but having "official one" could greatly promote Nini use in various kinds of projects, and possibly draw more contributors.

**Comment by bmatzelle** [comment]
I totally agree.  I've been meaning to get to it for some time.  I haven't ever created one before so I'll need to learn how first.

**Comment by kf6kjg** [comment]
@bmatzelle Any motion on that front?  Anything the community, including myself, can help with?

**Comment by bmatzelle** [comment]
No, I haven't done anything but I would appreciate the help.
